docs/troubleshooting.md

```diff
@@ -149,8 +149,9 @@ to use the same versions as OpenTelemetry .NET Automatic Instrumentation.
 A simple way to ensure that no such conflicts happen is to add the
 `OpenTelemetry.AutoInstrumentation` package to your application.
 For instructions about how to add it to your application, see
-[Using the OpenTelemetry.AutoInstrumentation NuGet packages](./using-the-nuget-packages.md#using-the-opentelemetryautoinstrumentation-nuget-packages)
-.
+[Using the OpenTelemetry.AutoInstrumentation NuGet packages](./using-the-nuget-packages.md#using-the-opentelemetryautoinstrumentation-nuget-packages). 
+This is the recommended way to deploy automatic instrumentation, but it can't
+be used in all scenarios.
```

Comment:
Question about the preceding paragraph:

> To handle dependency versions conflicts,
update the instrumented application's project references
to use the same versions as OpenTelemetry .NET Automatic Instrumentation.

Does this describe how the NuGet packages work or is it describing a manual alternative to NuGet? Depending on which it is, I think we could rearrange things a bit to present the solution sooner.

Reply:
You can use either
- our, unified packages. It will take care about all dependency versions
- manually cleanup all conflicts, there is no need to use our package. A lot of manual work.